packages/insomnia/src/ui/components/codemirror/one-line-editor.tsx

```diff
@@ -109,7 +109,7 @@ export const OneLineEditor = forwardRef<OneLineEditorHandle, OneLineEditorProps>
     codeMirror.current.on('beforeChange', (_: CodeMirror.Editor, change: CodeMirror.EditorChangeCancellable) => {
       const isPaste = change.text && change.text.length > 1;
       if (isPaste) {
-        if (change.text[0].startsWith('curl')) {
+        if (change.text[0].startsWith('curl') || !change.text.join('').trim()) {
```

Comment:
Thanks @git-commit-amen the intent could be a bit clearer though.
```suggestion
        const startsWithCurl = change.text[0].startsWith('curl');
        const isWhitespace = change.text.join('').trim();
        if (startsWithCurl || !isWhitespace) {
```

Reply:
Good pickup - yeah I agree. Will keep clarity in mind for any future commits. Thanks!